src/expire.c

```diff
@@ -489,6 +489,47 @@ int checkAlreadyExpired(long long when) {
     return (when <= mstime() && !server.loading && !server.masterhost);
 }
 
+#define FLAG_NO_FLAGS 0
```

Comment:
`FLAG_NO_FLAGS` is never used. Do we need it?

Reply:
updated to use it as initial value of `flags` following `t_string.c`